pkg/rpm/postupgrade

```diff
@@ -175,15 +184,24 @@ docker run --rm -d --name $SVC_NAME -p 9200:9200 \
   su elastic -c "$ELASTIC_BIN -Ecluster.initial_master_nodes=$NODE_NAME \
   -Enode.name=$NODE_NAME -Ecluster.name=$CLUSTER_NAME"
 
-if check_elasticsearch; then
-    report "SUCCESS" "Container started within timeout"
+#
```

Comment:
1) Preparing the host for data import to new elasticsearch storage
2) Starting container with new elasticsearch
Shouldn't run on the delegate host